### DISCUSSION TITLE
How do you organize your Speakers

### DISCUSSION THREAD

**Discussion by JDogg016** [music-assistant/support#3109] [Q&A] [answered]
Let's get the obvious out of the way first: bravo on an exceptionally cool piece of software! It has made it very easy to automate music and announcements within the Home Assistant framework, and I couldn't be happier. 

That being said, I have a lot of questions about speaker organization. I have Sonos speakers, as well as HomePods that are all already integrated into Home Assistant through HomeKit and through the Sonos integration. Music Assistant now creates its own media players to manage the devices as well.

So my question for the group is what sort of advice or tips can everyone offer up to manage the music assistance, speakers, group speakers within Music Assistant, and the individual Sonos and HomePod speakers? 

I find myself struggling from time to time when I create automations because I have to go back and remember which living room Sonos belongs to what integration. 

Also, what is the impact if I simply remove the Sonos integration altogether and rely only on Music Assistant or remove the HomePods through the HomeKit integration from Home Assistant?

**Comment by OzGav** [comment, marked as answer]
If you are happy with MA then yes, remove the overlapping HA integrations. 

You can rename entities in HA. I have added `ma_` to the start of all of mine.

Note that Sonos accepts AirPlay so if you use that you will be able to group your Sonos and HomePods together (if you wanted to)

**Reply by JDogg016** [reply]
So I can safely remove the Sonos and HomePod integrations. 

As for MA, would I then want to disable all the Sonos players that are also AirPlay2 Compatible. I have a few older Sonos devices (ie Sonos Connect)I don't think can be grouped with because they're only AirPlay 1.

**Reply by OzGav** [reply]
No they should all be backwards compatible with AirPlay 1. MA only uses AirPlay 1

**Reply by JDogg016** [reply]
Very good. Hopefully last question. Why would I need Sonos as a player at all if I can run everything through airplay?

Is there an advantage to having both?

**Reply by OzGav** [reply]
That is kind of up to you. Look at the differences here https://music-assistant.io/player-support/

Additionally Sonos has native announcement support so that will be nicer to use.https://music-assistant.io/integration/announcements/#native-support
